modules/market/src/main/java/com/opengamma/strata/market/curve/JacobianCurveCalibration.java

```diff
@@ -29,49 +27,41 @@
 import org.joda.beans.impl.direct.DirectMetaPropertyMap;
 
 import com.google.common.collect.ImmutableList;
-import com.opengamma.strata.market.curve.CurveName;
+import com.opengamma.strata.market.curve.definition.CurveParameterSize;
+import com.opengamma.strata.math.impl.matrix.DoubleMatrix2D;
 
 /**
- * Information about a block of curves that was built together.
+ * Jacobian matrix information produced during curve calibration.
  * <p>
- * This provides details of the names of the curves and the number of parameters they have.
- * This is used in curve calibration to create and split a single array of all parameters from all curves.
+ * The inverse Jacobian matrix produced using curve calibration is stored here.
+ * The information is used to calculate market quote sensitivity.
  */
 @BeanDefinition(builderScope = "private")
-public final class CurveBuildingBlock
+public final class JacobianCurveCalibration
```

Comment:
I'm not sure about the class name. A curve calibration isn't something you'd expect to be represented as an object. How about `CurveJacobian` or `CurveCalibrationJacobian`? Or, given we're in the `curve` package, just `Jacobian`?